x-pack/plugin/core/src/main/resources/monitoring-beats-mb.json

```diff
@@ -0,0 +1,2198 @@
+{
+  "index_patterns": [".monitoring-beats-${xpack.stack.monitoring.template.version}-*"],
```

Comment:
This is the same pattern as https://github.com/elastic/elasticsearch/blob/master/x-pack/plugin/core/src/main/resources/monitoring-es.json#L2 - are we planning to introduce a new variable to set the mb version to `10`?

Reply:
Currently the `xpack.stack.monitoring.template.version` is set to version 8 for the metricbeat indices. We talked it over a bit more and agreed that 8 was indeed the more appropriate version number to put here. The monitoring-es.json file uses the `xpack.monitoring.template.version` property (doesn't contain `stack`)

Reply:
ah... `xpack.monitoring.template.version` vs `xpack.stack.monitoring.template.version` - I missed the distinction. Thanks!